flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/nodes/CygnusApplication.java

```diff
@@ -124,14 +166,14 @@ public static void main(String[] args) {
             boolean reload = !commandLine.hasOption("no-reload-conf");
 
             if (commandLine.hasOption('h')) {
-                new HelpFormatter().printHelp("flume-ng agent", options, true);
+                new HelpFormatter().printHelp("cygnus-flume-ng agent", options, true);
```

Comment:
Yes, simply this help line was not updated. But the README explains that we have a `cygnus-flume-ng` specific script.